tests/bootstrap_tests/test_configuration_writer.py

```diff
@@ -29,7 +29,9 @@ def _write_mock_config(self, shotgun_yml_data=None):
         """
         Creates a fake config with the provided shotgun.yml data.
         """
-        mock_config_root = os.path.join(self.tank_temp, "template", self.id())
+        # Make the file name not too long or we'll run into file length issues on Windows.
```

Comment:
Turns out the tests were silently failing on Windows because batch files do not abort on first process failure. We were running into path length issues.